src/main/resources/openapi/NotificationsAPIs.yaml

```diff
@@ -2327,11 +2330,17 @@ components:
           type: integer
         hasSentNotRegisteredToVoteCommunication:
           type: boolean
+        numSignatureRequestCommsSent:
```

Comment:
In the above properties, Communications is written out in full, which seems inconsistent with the new names. Not sure whether a change would be better or worse, as the properties are quite long as is, but thought I'd mention